src/partition/repart.c

```diff
@@ -2116,6 +2176,7 @@ static int partition_read_definition(Partition *p, const char *path, const char
                 { "Partition", "ExcludeFiles",             config_parse_exclude_files,     0,                                  &p->exclude_files_source    },
                 { "Partition", "ExcludeFilesTarget",       config_parse_exclude_files,     0,                                  &p->exclude_files_target    },
                 { "Partition", "MakeDirectories",          config_parse_make_dirs,         0,                                  &p->make_directories        },
+                { "Partition", "MakeSymlinks",             config_parse_make_symlinks,     0,                                  &p->make_symlinks           },
```

Comment:
btw, we should probably rename Subvolume= to MakeSubvolumes=, no?

Reply:
I'm not sure, only if we make `MakeSubvolumes=` actually create the directories I would say

Reply:
ah, right. we probably should though.

Reply:
I'm not a big fan. I know `MakeDirectories=` doesn't do anything fancy like permissions and ownership and whatnot, but if it did, we'd have to duplicate all of that to `MakeSubvolumes=`. I prefer having `MakeDirectories=` be about creating directory inodes and `Subvolumes=` just about turning directory inodes into subvolumes